Cargo.toml

```diff
@@ -22,3 +26,4 @@ winter_utils = { version = "0.4.1", package = "winter-utils", default-features =
 [dev-dependencies]
 proptest = "1.0.0"
 rand_utils = { version = "0.4", package = "winter-rand-utils" }
+criterion = "0.4"
```

Comment:
We often use some form of reports in criterion. Before `0.4`, it was included automatically in case you want to generate only parseable csv files with the collected data, so maybe we want to include it here

`criterion = { version = "0.4", features = ["html_reports"] }`

Reply:
We haven't made use of any reports yet. I think it is fine to omit this feature for now and add it later once we actually start using reports.